elide-datastore/elide-datastore-hibernate5/src/main/java/com/yahoo/elide/datastores/hibernate5/HQLTransaction.java

```diff
@@ -129,7 +129,7 @@ else if (filters != null) {
             }
 
             // add sorting into HQL string query generation
-            if (!sortingRules.isEmpty()) {
+            if (sortingRules != null && !sortingRules.isEmpty()) {
```

Comment:
Passing null is still a breaking change in some sense.  I don't understand why we need to change the DataStores at all (unless there was a bug).